src/main/java/MealDemo/entity/Frequency.java

```diff
@@ -8,7 +8,7 @@ public class Frequency {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     @Column(name="id")
-    private int id;
+    private long id;
```

Comment:
Not related to the PR but something to add to your todo list: What is the difference between `long` and `Long` or `int` and `Integer`?